go/vt/vttablet/endtoend/config_test.go

```diff
@@ -203,22 +203,23 @@ func testQueryPlanCache(t *testing.T, cachedPlanSize, cachePlanSize2 int) {
 	client := framework.NewClient()
 	_, _ = client.Execute("select * from vitess_test where intval=:ival1", bindVars)
 	_, _ = client.Execute("select * from vitess_test where intval=:ival2", bindVars)
-	time.Sleep(100 * time.Millisecond)
+	assert.Equal(t, 1, framework.Server.QueryPlanCacheLen())
 
 	vend := framework.DebugVars()
-	verifyIntValue(t, vend, "QueryCacheLength", 1)
+	// verifyIntValue(t, vend, "QueryCacheLength", 1)
```

Comment:
did you leave this commented intentionally?

Reply:
Nope my bad